skycoin/skywire
# Improvements for how the manager gets the data

Did you run `make format && make check`?
The go code was not changed. `npm run lint` and `npm run build` were used.

Fixes #446

 Changes:	
- The manager now gets the visor data by calling the `GET api/visors/{nodeKey}/summary` API endpoint, instead of making several API requests. This makes the manager load the visor detail pages faster.
- The objects used for representing the server responses were changed, as the codes was previously imitating the exact objects returned by the API and that causes problems which propagates fast to several parts of the app if changes are made to the API.
- The code for showing the route details was simplified.

How to test this PR:
Select a visor in the visor list of the Skywire manager, the details page should open faster than before. Also open the details of a route, the data should be displaying immediately, without loading time.
